appengine/mailgun/README.md

```diff
@@ -0,0 +1,64 @@
+## Mailgun on Google App Engine
+
+> [Mailgun](https://www.mailgun.com/): The Email Service For Developers
+
+### Sign up for Mailgun
+
+1. Sign up for a [Mailgun account](https://mailgun.com/signup).
+1. Add a [new domain](https://mailgun.com/app/domains).
+1. Find your API key in your new domain's settings.
+
+### Create a new Node.js app
+
+This example uses [Express.js](http://expressjs.com) and
+[node-mailgun](http://github.com/shz/node-mailgun).
+
+### Configure
+
+Create an `app.yaml` in the root of your application with the following
+contents:
+
+```yaml
+runtime: nodejs
+vm: true
+api_version: 1
+env_variables:
+  PORT: 8080
+  MAILGUN_API_KEY: <your-mailgun-api-key>
+```
+
+### Start the app locally
+
+```
+$ MAILGUN_API_KEY=<your-mailgun-api-key> npm start
+```
+
+Now visit http://localhost:8080 and try sending yourself an email.
+
+When the app is deployed to Google Cloud Platform the `MAILGUN_API_KEY`
+environment variable will automatically be supplied to the app.
```

Comment:
Replace `will automatically be supplied to the app` with `will be set to the value specified in app.yaml`.